ftrackhq/integrations
# fix: framework build with development works as expected

Resolves : 

* CLICKUP-https://app.clickup.com/t/862kgfbh7
* FTRACK-
* SENTRY-
* ZENDESK-


This PR has been tested on :



## Changes
- Poetry is capable to build lib/framework-core with the development dependnecies.
-


## Test
- Go to the libs/framework-core folder of the monorepo in the shell interpreter
- Create a new py3.7 environment
- Make sure you don't have the poetry.lock file
- `poetry install --with development`
- If you check the installed packages it should point to the development ones and if you start a python should be able to execute the framework with no errors:
    - `from ftrack_framework_core import host`
- Remove your environment and create a new one. Make sure you also removed the poetry.lock file
- `poetry install`
- This should now fail because the libraries are not yet published to pypi.
- `poetry update` should also be working to update dependencies from dev to pypi without removing the poetry.lock file but I didn't test it. https://python-poetry.org/docs/cli#update
